src/main/java/com/bfrisco/benchants/enchants/Trench.java

```diff
@@ -30,33 +33,43 @@ public Trench() {
     @EventHandler
     @SuppressWarnings("unused")
     public void onBlockBreakEvent(BlockBreakEvent event) {
-        if (IGNORE_LOCATIONS.contains(event.getBlock().getLocation())) {
+       if (IGNORE_LOCATIONS.contains(event.getBlock().getLocation())) {
             IGNORE_LOCATIONS.remove(event.getBlock().getLocation());
             return;
         }
-
+        Player player = event.getPlayer();
         ItemStack item = event.getPlayer().getInventory().getItemInMainHand();
-        if (!hasTrench(item)) return;
+//*****************Proposed new check, removes nbtapi dependency*********************************
+        if (!ItemInfo.isTitanTool(item)) return;
+        if (!ItemInfo.isActive(item)) return;
+        if (item.getType() == pick) {
```

Comment:
I do not think this check if it is a pick should not be needed here.

In the configuration we specify which items can be enchanted - right now, Netherite pick is in the configuration, and this would break its functionality for netherite pickaxes or any other items in the configuration

Reply:
The concept will eventually be an assignment of enchants based on tool type + ancient power lore. This isn't fully hashed out yet but block break events will occur when the axe enchantment is implemented and the way the enchantment will be assigned will be through this check.

Reply:
Got it. In that case, I wonder if there is a better way to tell if it is *any* kind of pick. If not, we may want to store materials in another ArrayList similar to the `IMBUED_LORE` and perform checks that way. Same thing for axes, etc.

Reply:
yeah i was basically thinking another list luckily we will only have diamond and netherite variations with the tools as they come diamond already and can only be upgraded to netherite